app/Http/Kernel.php

```diff
@@ -1,10 +1,9 @@
-<?php
-
-namespace App\Http;
+<?php namespace App\Http;
 
 use Illuminate\Foundation\Http\Kernel as HttpKernel;
 
 class Kernel extends HttpKernel
+
```

Comment:
Opening brace of a class must be on the line following the class declaration; found 1 line(s)